app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -74,18 +78,7 @@ class SmokeTest {
     private var bookmarksListIdlingResource: RecyclerViewIdlingResource? = null
     private var localeListIdlingResource: RecyclerViewIdlingResource? = null
     private val customMenuItem = "TestMenuItem"
-
-    // This finds the dialog fragment child of the homeFragment, otherwise the awesomeBar would return null
-    private fun getAwesomebarView(): View? {
-        val homeFragment = activityTestRule.activity.supportFragmentManager.primaryNavigationFragment
-        val searchDialogFragment = homeFragment?.childFragmentManager?.fragments?.first {
-            it.javaClass.simpleName == "SearchDialogFragment"
-        }
-        return searchDialogFragment?.view?.findViewById(R.id.awesome_bar)
-    }
-

```

Comment:
Just double checking here if this is refactored elsewhere?

Reply:
wasn't used, I think because Sebastian changed the tests with the new compose awesome bar